SPECS/zfs-kmod.spec

```diff
@@ -51,7 +51,7 @@
 
 Name:           %{module}-kmod
 
-Version:        0.8.5
+Version:        2.1.14
 Release:        2%{?dist}
```

Comment:
Should it be back to 1?

Reply:
Mmmm yes I think that would be logical. I will do that.